src/Stratis.Bitcoin/Base/ChainHeadersBehavior.cs

```diff
@@ -33,62 +30,23 @@ public class ChainHeadersBehavior : NetworkPeerBehavior
         /// <summary>Provider of IBD state.</summary>
         private readonly IInitialBlockDownloadState initialBlockDownloadState;
 
-        /// <summary><c>true</c> if the chain should be kept in sync, <c>false</c> otherwise.</summary>
-        public bool CanSync { get; set; }
-
-        /// <summary><c>true</c> to sync the chain as headers come from the network, <c>false</c> not to sync automatically.</summary>
-        public bool AutoSync { get; set; }
-
         /// <summary>
-        /// Our view of the peer's headers tip constructed on peer's announcement of its tip using "headers" message.
-        /// <para>
-        /// The announced tip is accepted if it seems to be valid. Validation is only done on headers
-        /// and so the announced tip may refer to invalid block.
-        /// </para>
+        /// Our view of the peer's consensus tip constructed on peer's announcement of its tip using "headers" message.
         /// </summary>
-        /// <remarks>It might be different than concurrent's chain tip, in the rare event of large fork > 2000 blocks.</remarks>
-        private ChainedHeader pendingTip;
-
-        /// <summary>Information about the peer's announcement of its tip using "headers" message.</summary>
-        public ChainedHeader PendingTip
-        {
-            get
-            {
-                ChainedHeader tip = this.pendingTip;
-                if (tip == null)
-                    return null;
-
-                // Prevent memory leak by returning a block from the chain instead of real pending tip if possible.
-                return this.Chain.GetBlock(tip.HashBlock) ?? tip;
-            }
-        }
-
-        /// <summary><c>true</c> to respond to "getheaders" messages, <c>false</c> to ignore it.</summary>
-        public bool CanRespondToGetHeaders { get; set; }
+        /// <remarks>
+        /// The announced tip is accepted if it seems to be valid. Validation is only done on headers and so the announced tip may refer to invalid block.
+        /// </remarks>
+        public ChainedHeader ExpectedTip { get; private set; }
```

Comment:
Should this not be `PeerTip`?

Reply:
That's expected peer tip. 
We were arguing with apro on what name suits best, ended up with ExpectedTip. 
I can rename to ExpectedPeerTip if you'd like.

Reply:
Ok if there was an argument then I will just stay out of it leave as is.

Reply:
Not really an argument, just didn't know what would be the name. 

ExpectedPeerTip sounds good too, will fix.